censys/cli.py

```diff
@@ -535,42 +535,66 @@ def cli_config(_):  # pragma: no cover
     Args:
         _: Argparse Namespace.
     """
-
-    api_id_prompt = "Censys API ID"
-    api_secret_prompt = "Censys API Secret"
-
     config = get_config()
-    api_id = config.get(DEFAULT, "api_id")
-    api_secret = config.get(DEFAULT, "api_secret")
 
-    if api_id and api_secret:
-        redacted_id = api_id.replace(api_id[:32], 32 * "*")
-        redacted_secret = api_secret.replace(api_secret[:28], 28 * "*")
-        api_id_prompt = f"{api_id_prompt} [{redacted_id}]"
-        api_secret_prompt = f"{api_secret_prompt} [{redacted_secret}]"
+    api_choice = input("Please choose an API type to configure: Search or ASM: ")
```

Comment:
Instead of modifying the existing  CLI, would you be able to make a `store_true` flag instead? Or possibly another subcommand.